Kernel/Jail.cpp

```diff
@@ -26,7 +75,7 @@ void Jail::detach(Badge<Process>)
         VERIFY(my_attach_count > 0);
```

Comment:
not sure if itʼs correct—you could probably stil have a perfectly valid interleave here; maybe just bailing out with `return` instead?

Reply:
The idea is that once all processes detach from a jail then it will be destroyed, and when we detach the attach count should slways be greater than 0. Just look where we call this thing.

Reply:
read through the whole thing once again—less doubt now—thanks for the explanation! 🙇🏻‍♂️